src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -362,207 +362,183 @@ public void ModifyMethod_WithTuples()
         [Fact]
         public void ModifyMethod_WithAttributes1()
         {
-            var source0 =
-@"class C
+            using var _ = new EditAndContinueTest(options: TestOptions.DebugExe, targetFramework: TargetFramework.NetStandard20)
+                .AddGeneration(
+                    source: @"
+class C
 {
     static void Main() { }
     [System.ComponentModel.Description(""The F method"")]
     static string F() { return null; }
-}";
-            var source1 =
-@"class C
+}",
+                    verification: g =>
+                    {
+                        g.VerifyTypeDefNames("<Module>", "C");
+                        g.VerifyMethodDefNames("Main", "F", ".ctor");
+                        g.VerifyMemberRefNames(/*CompilationRelaxationsAttribute.*/".ctor", /*RuntimeCompatibilityAttribute.*/".ctor", /*Object.*/".ctor", /*DebuggableAttribute*/".ctor", /*DescriptionAttribute*/".ctor");
```

Comment:
>VerifyMemberRefNames [](http://example.com/codeflow?start=26&length=20)

As a follow up - we should be able to resolve fully qualified names of MemberRefs, MethodDefs and TypeDefs.